index.html

```diff
@@ -997,41 +978,34 @@ <h2>
         changes:
       </p>
       <pre class="example">
-        const controller = new AbortController();
-        const signal = controller.signal;
-
         const checkbox = document.createElement("input");
         checkbox.setAttribute("type", "checkbox");
         document.body.appendChild(checkbox);
 
         const lock = new WakeLock("screen");
-        lock.onactivechange = () => {
-          checkbox.checked = lock.active;
-        };
-        lock.request({ signal });
+        lock.onactivechange = () => checkbox.checked = lock.active;
+        lock.request();
```

Comment:
I imagine a real "check this box to enable a wake lock" code would work something like this:

```
lock.onactivechange = () => checkbox.checked = lock.active;
checkbox.onchange = async () => {
  try {
    checkbox.disabled = true;
    await lock.request();
  } catch (e) {
    checkbox.checked = false;
  } finally {
    checkbox.disabled = false;
  }
};
```

This might be too complicated for this particular example but I notice that none of the examples handle the case where the Promise returned by `request()` rejects.

Reply:
good suggestion